docs/learn/smart-contract-developers/create-nft-collection-with-lsp7-tokenId.md

```diff
@@ -0,0 +1,956 @@
+
+## Use Case
+
+Let's take the example of [The Dematerialised](https://thedematerialised.com), and more precisely their KLxENDLESS PHYSITAL COLLECTION. As you can see in the picture below, we have three medaillons of different colors.
+
+![KLxENDLESS PHYSITAL COLLECTION](../../../static/img/learn/klxendless-physital-collection.png)
+
+Each medaillon has a supply limit and each medaillon collection has its own metadata with different attributes. What we will do is create an [LSP8 collection](../../standards/tokens//LSP8-Identifiable-Digital-Asset.md) that will represent the KLxENDLESS PHYSITAL collection and each medaillon a sub-collection (purple, blue or gold) will be represented by a limited supply [LSP7 token](../../standards/tokens//LSP7-Digital-Asset.md).
+
+## Prerequisites
+
+- [Hardhat installed and initialized](./getting-started.md)
+- [`@lukso/lsp-smart-contracts`](https://www.npmjs.com/package/@lukso/lsp-smart-contracts) package installed from npm using `npm i @lukso/lsp-smart-contracts@0.14.0` (or the latest version)
+- [`@erc725/erc725.js`](https://www.npmjs.com/package/@erc725/erc725.js) package installed from npm using `npm i @erc725/erc725js` (we will use it to encode the token metadata)
+
+## Create the LSP7 token
+
+Even though it seems a little bit counter intuitive, we will start by creating the LSP7 token. The reason is that the LSP8 collection will be in charge of deploying the LSP7 sub-collection tokens.
+
+### Specify what needs to be set at deployment
+
+To create the LSP7 token contract, we will need to know what we want to be set at deployment. In our case, we want to set the following:
+
+- the name of the sub-collection
+- the symbol of the sub-collection
+- the address of the owner of the sub-collection (in our case it will the the LSP8 collection contract)
+- the initial receiver of the tokens that will be minted at deployment (could be the token creator)
+- the number of tokens that can be minted
+- whether the token is divisible or not
+- the total supply of the token
+- the [Token Type](../../standards/tokens/LSP4-Digital-Asset-Metadata.md#types-of-digital-assets)
+- the [LSP4 Metadata](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md#lsp4metadata)
+
+This is just an example but you may want to set a deployment of other parameters such as the [LSP4 Creators array](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md#lsp4creators).
+
+### Imports
+
+Let's start creating an `LSP7Token.sol` file in the `contracts` folder and importing:
+
+- the [`LSP7DigitalAsset.sol`](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP7DigitalAsset/LSP7DigitalAsset.sol) contract
```

Comment:
Not sure if we need the `.sol` extension, since we are talking about importing a `contract` here?

```suggestion
- the [`LSP7DigitalAsset`](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP7DigitalAsset/LSP7DigitalAsset.sol) contract
```